### ISSUE TITLE
Ubuntu Startproblem

### ISSUE THREAD

**Issue by kashperle** [Torridity/dsworkbench#59]
./DSWorkbench 
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fb971c23009, pid=4229, tid=4261
#
# JRE version: OpenJDK Runtime Environment (9.0) (build 9-internal+0-2016-04-14-195246.buildd.src)
# Java VM: OpenJDK 64-Bit Server VM (9-internal+0-2016-04-14-195246.buildd.src, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libjava.so+0x1d009]  JNU_GetEnv+0x19
#
# Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P" (or dumping to /home/user/DS-Workbench/core.4229)
#
# An error report file with more information is saved as:
# /home/user/DS-Workbench/hs_err_pid4229.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Abgebrochen (Speicherabzug geschrieben)

**Comment by extremeCrazyCoder** [comment]
https://bugs.openjdk.java.net/browse/JDK-8159050
sieht erstmal nach einem generellen Problem aus mit deiner Java Version
du könntest die Oracle Java Binaries verwenden. (https://www.java.com/de/download/)

nachdem ich selber auch Ubuntu verwende (18.04 / 17.10) ist es eher kein genereller Bug. Ich werde mal versuchen das nachzustellen. Ich melde mich dann wieder

**Comment by kashperle** [comment]
openjdk version "1.8.0_181"
OpenJDK Runtime Environment (build 1.8.0_181-8u181-b13-0ubuntu0.16.04.1-b13)
OpenJDK 64-Bit Server VM (build 25.181-b13, mixed mode)

damit läufts

**Comment by extremeCrazyCoder** [comment]
danke fürs probieren mit einer anderen Version

ich konnte es schon nachstellen. Nachdem ich gelesen habe, dass es an JOptionPane liegt und es mit einer anderen Java Version eh funktioniert werde ich dafür keinen fix schreiben.
JOptionPane wird im Code sehr oft verwendet (für alle Abfragen an den User & Fehlermeldungen; eine suche ergab etwas über 400 verwendungen), sodass es ziemlich aufwändig wäre auf diese Library zu verzichten.

Gibt es bei dir irgendeinen Grund, dass du java 1.9 installiert hattest? Zumindest bei mir war im Paket default-jre java 1,8 enthalten womit es funktioniert.
Solltest du java 1.9 als default brauchen so könnte ich dir anbieten, eine Version mit embedded java zusammen zu bauen

**Comment by kashperle** [comment]
nope, kein Grund, kann sein das es bei mir irgendeine Paketquelle mitbringt.

**Comment by extremeCrazyCoder** [comment]
@Torridity das hier dürfte auch erledigt sein
